app/backend/src/tests/test_fixtures.py

```diff
@@ -335,3 +325,19 @@ def testconfig():
 
     config.clear()
     config.update(prevconfig)
+
+
+@pytest.fixture
+def fast_passwords():
+    # password hashing, by design, takes a lot of time, which slows down the tests. here we jump through some hoops to
+    # make this fast by replacing the hashing algo with a fast (non-password) hash, blake2b
+
+    def fast_verify(hashed: bytes, password: bytes) -> bool:
+        return hashed == blake2b(password)
+
+    def fast_hash(password: bytes) -> bytes:
+        return blake2b(password)
```

Comment:
I would just do `return b'hashof:' + password`, no need to use a real hashing algorithm in the tests.

Reply:
Hmmm... I guess.